dask/dataframe/groupby.py

```diff
@@ -302,8 +302,9 @@ def _var_agg(g, levels, ddof):
     g = g.groupby(level=levels, sort=False).sum()
     nc = len(g.columns)
     x = g[g.columns[: nc // 3]]
-    x2 = g[g.columns[nc // 3 : 2 * nc // 3]].rename(columns=lambda c: c[:-3])
-    n = g[g.columns[-nc // 3 :]].rename(columns=lambda c: c[:-6])
+    # chunks columns are tuples (number, str), so we just keep the number part
```

Comment:
Small nitpick: in general `c` can be non-numeric (e.g. string columns)

```suggestion
    # chunks columns are tuples (value, name), so we just keep the value part
```